titanium-scanner.xcodeproj/project.pbxproj

```diff
@@ -278,8 +281,9 @@
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
 				INSTALL_PATH = /usr/local/lib;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LIBRARY_SEARCH_PATHS = "";
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
Is this setting really required? I usually don't use it.

Reply:
idk what happened. just opened the project with Xcode